modules/statistics/js/table_statistics.js

```diff
@@ -16,12 +16,12 @@ $('#showVL').click(
         }
     }
 );
-
-$(document).ready(
-    function(){
-        $("#bigtable").DynamicTable({ "freezeColumn" : "tpcol" });
-    }
-);
+//freezecolumn not sufficient for complex tables
```

Comment:
I would just remove this code instead of commenting it out.

Reply:
i left it in there in case someone in the future requests for this to be implemented and doesn't realize that it was previously removed for a reason

Reply:
I agree with Zain, there should be commented out code anywhere.